src/test/java/org/folio/rest/impl/MappingProfileServiceTest.java

```diff
@@ -14,12 +12,23 @@
 import org.junit.runner.RunWith;
 import org.mockito.junit.jupiter.MockitoExtension;
 
+import io.restassured.RestAssured;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import io.vertx.junit5.VertxExtension;
+
 @RunWith(VertxUnitRunner.class)
 @ExtendWith(MockitoExtension.class)
 @ExtendWith(VertxExtension.class)
 class MappingProfileServiceTest extends RestVerticleTestBase {
 
   private static final String INVALID_TRANSFORMATION = "123qq$qweqwe";
+  private static final String INVALID_TAG = "12a  $a";
+  private static final String INVALID_FIRST_INDICATOR = "123. $a";
+  private static final String INVALID_SECOND_INDICATOR = "123 ,$a";
+  private static final String INVALID_SUBFIELD = "123  $1234";
+  private static final String EMPTY_TRANSFORMATION = "";
+  private static final String VALID_TRANSFORMATION = "13qq$q";
```

Comment:
unused field

Reply:
Corrected, I've replaced the old variable with a new one which is used.